<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>MOSAIC Lab</title>
  <link rel="stylesheet" href="styles.css">
</head>

<script>
document.addEventListener("DOMContentLoaded", function () {
  const toggle = document.getElementById("theme-toggle");
  const root = document.documentElement;

  const saved = localStorage.getItem("theme");
  if (saved === "light") {
    root.setAttribute("data-theme","light");
    toggle.textContent = "‚òÄÔ∏è";
  } else {
    toggle.textContent = "üåô";
  }

  toggle.addEventListener("click", () => {
    const lightMode = root.getAttribute("data-theme") === "light";
    if (lightMode) {
      root.removeAttribute("data-theme");
      toggle.textContent = "üåô";
      localStorage.setItem("theme","dark");
    } else {
      root.setAttribute("data-theme","light");
      toggle.textContent = "‚òÄÔ∏è";
      localStorage.setItem("theme","light");
    }
  });
});
</script>

<body>
<div class="container">

<header class="site-header">
    <div class="brand">
  <img src="Mosaic.png" class="logo-img" alt="MOSAIC logo">

  <div class="brand-text">
    <h1>Lab</h1>
    <p>Molecular Omics, Structure &amp; Image Computing</p>
  </div>
</div>

  <nav class="header-nav">
    <a href="#projects">Projects</a>
    <a href="#members">Members</a>
    <a href="#alumni">Alumni</a>
    <a href="#collaborations">Collaborations</a>
    <a href="#publications">Publications</a>
    <a href="#join">Join</a>
    <a class="contact-link" href="mailto:msplaksha@gmail.com">Contact</a>
    <button id="theme-toggle" class="theme-toggle">üåô</button>
  </nav>
</header>

<section class="hero">
  <div class="hero-card">
    <span class="eyebrow">Integrating Omics ¬∑ Structure ¬∑ Imaging</span>
    <h2>MOSAIC Lab: Molecular Omics, Structure &amp; Image Computing</h2>
    <p class="lead">We unify molecular omics, structural biology, and advanced image computing to understand biological systems across scales. Our research integrates protein structure, multiomics layers, and computational modeling to address India&#x27;s pressing challenges in health.</p>
    <div class="cta-row">
      <a class="btn" href="#join">Join MOSAIC</a>
    </div>
    <p class="hero-mission"><strong>Our Mission:</strong> We are building science in both bottoms up and top downs manner, centering proteins, their encoding genes and mRNA sequences, and modern computational imaging to decode molecular systems from atoms to phenotypes. We aim to accelerate simulating biological phenomenon to have real-world impact by developing integrative frameworks for rare and common lifestyle diseases relevant to India.</p>
  </div>

  <aside class="side-panel">
    <div class="panel-title"><b>What We Build:</b></div>
    <div class="panel-pills"><div class='pill'>Integrating omics, structure, and imaging</div> <div class='pill'>Top-down and bottom-up biological inference</div> <div class='pill'>AI-driven molecular and phenotypic interpretation</div> <div class='pill'>Translational pipelines customized to cater health of Indians</div></div>

    <div class="panel-title" style="margin-top:16px;"><b>Open Roles</b></div>
    <div>Stay Tuned....</div>
                         <br></br>

    <div class="panel-footer">
      <a class="pill" href="https://github.com/msplaksha">GitHub ‚Üí</a>
    </div>
  </aside>
</section>

<section id="projects">
  <h2 class="section-title">Research Projects</h2>
  <div class="grid">
      <article class="card">
        <span class="tag">Structural Dynamics</span>
        <h3 class="card-title">Blood‚ÄìBrain Barrier Regulation &amp; Transporter Dynamics</h3>
        <p class="card-body">Collectively studying ligand binding, conformational dynamics, and molecular simulations of transporters, we aim to understand how they regulate metabolite flux across the blood‚Äìbrain barrier.</p>
      </article>
    
      <article class="card">
        <span class="tag">Neuro-Omics</span>
        <h3 class="card-title">Biomarker Discovery for Amyotrophic Lateral Sclerosis (ALS)</h3>
        <p class="card-body">Using integrative multi-omics (genomic, transcriptomics &amp; methylomics) and computational pipelines, we aim to identify early-stage ALS biomarkers, enabling improved diagnosis, risk stratification, and personalized therapeutic strategies.</p>
      </article>
    
      <article class="card">
        <span class="tag">Cardiac-Omics</span>
        <h3 class="card-title">Biomarkers for Acute Coronary Syndrome Through Integrated Multi-Omics</h3>
        <p class="card-body">Combining real-world patient data (from northern India) with transcriptomic, proteomic, and metabolomic signatures, we aim to uncover molecular indicators of early cardiac stress and ACS progression.</p>
      </article>
    
      <article class="card">
        <span class="tag">Therapeutics</span>
        <h3 class="card-title">Hepcidin-Targeted Therapeutics for Anemia</h3>
        <p class="card-body">We are seeking personnel to develop therapeutic strategies to modulate hepcidin, the master regulator of systemic iron homeostasis, with aim to address anemia of inflammation and iron-restricted erythropoiesis.</p>
      </article>
    
      <article class="card">
        <span class="tag">AI Imaging</span>
        <h3 class="card-title">Deep Learning for Bone Marrow Histopathology</h3>
        <p class="card-body">We are building deep learning pipelines for automated interpretation of bone marrow biopsy images for informed detection of hematological abnormalities and advancement of diagnostic workflows.</p>
      </article>
    
      <article class="card">
        <span class="tag">Cancer Genomics</span>
        <h3 class="card-title">Curation of Germline Cancer Variants for Indian Populations</h3>
        <p class="card-body">We are currently mining literature and multi-omics datasets to compile germline cancer variant as well as transcriptomic proteomic sets. We aim in immediate future to use these in accordance to FAIR principles for enabling comparative molecular analyses between Indian cohorts and global populations.</p>
      </article>
    
      <article class="card">
        <span class="tag">Immune Structure</span>
        <h3 class="card-title">Structure-Driven Modeling of Cytokine‚ÄìReceptor Interactions</h3>
        <p class="card-body">We are modeling cytokine‚Äìreceptor assemblies using structural databases, involving state of art AlphaFold, docking, and thermodynamic profiling to inform ligand pleiotropy, and bioactivity. This will have contributions in enabling future of cytokine engineering.</p>
      </article>
    
      <article class="card">
        <span class="tag">Allostery</span>
        <h3 class="card-title">Allosteric Regulation in DgoR Transcriptional Repressor</h3>
        <p class="card-body">We have investigated structural and functional consequences of DgoR super-repressor mutations using long-timescale MD simulations, that have uncoverred allosteric pathways linked to DNA binding and effector-driven regulation. [See publication section]</p>
      </article>
    </div>
</section>

<section id="members">
  <h2 class="section-title">Our Team</h2>
  <div class="member-grid">
        <div class='pi-banner'>
            <img class='pi-banner-photo' src='../members/Monika.jpg' alt='Monika Sharma, Ph.D'>

            <div class='pi-banner-content'>
                <h2 class='pi-banner-name'>Monika Sharma, Ph.D</h2>
                <p class='pi-banner-role'>Principal Investigator</p>

                <p class='pi-banner-bio'>Group leader, who believes in leading by example. Her projects reflects the lab&#x27;s ethos, and drawing on her multidisciplinary background spanning chemistry, biology and computer science, she guides individual projects with blend of literature informed domain knowledge with strong data driven approaches.</p>

                <div class='pi-sections'>
            <div class='pi-section'>
                <h4>Education</h4>
                <ul><li>PhD in Computational Natural Sciences, IIIT Hyderabad, 2012</li></ul>
            </div>
            <div class='pi-section'>
                <h4>Experience</h4>
                <ul><li>Postdoctoral Fellow, Miichigan State University, USA: 2012-2013</li><li>Postdoctoral Fellow, NINDS, NIH, Bethesda, USA: 2013-2014</li><li>Indian Institute of Science Education and Research Mohali: 2015-2021</li><li>Plaksha University: 2021 onwards</li></ul>
            </div>
            <div class='pi-section'>
                <h4>Research Areas</h4>
                <ul><li>Protein structure analysis</li><li>Molecular dynamics, monte carlo and stochastic simulations</li><li>Systems-level multiomics</li><li>Machine learning for molecular biology</li></ul>
            </div></div>

                <div class='pi-links'>
                    <a class="mini-link" href="mailto:msplaksha@gmail.com"><svg viewBox='0 0 24 24'><path fill='currentColor'
d='M20 4H4a2 2 0 0 0-2 2v12a2 
2 0 0 0 2 2h16a2 2 0 0 0 
2-2V6a2 2 0 0 0-2-2zm0 4l-8 5-8-5V6l8 5 
8-5v2z'/></svg></a><a class="mini-link" href="#" target="_blank"><svg viewBox='0 0 24 24'><path fill='currentColor'
d='M19 3a2 2 0 0 1 2 2v14a2 2 0 0 
1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 
1 2-2h14M8 17v-7H6v7h2M7 
7.75A1.25 1.25 0 1 0 7 5a1.25 
1.25 0 0 0 0 2.75M18 
17v-4.5c0-2.25-2-3.25-3.5-3.25S12 
10.5 12 10.5V10H10v7h2v-4.2c0-1.25.75-1.8 
1.75-1.8S15.5 11.55 15.5 12.8V17h2z'/></svg></a><a class="mini-link" href="#" target="_blank"><svg viewBox='0 0 24 24'><path fill='currentColor'
d='M12 3L4 6v12a1 1 0 0 
0 1 1h14a1 1 0 0 0 1-1V6l-8-3zm6 
15H6V8.5l6-2.25L18 8.5V18z'/></svg></a><a class="mini-link" href="#" target="_blank"><svg viewBox='0 0 24 24'><path fill='currentColor'
d='M12 2a10 10 0 1 0 
0 20a10 10 0 0 0 0-20zm0 
4a6 6 0 1 1 0 12a6 6 0 0 1 
0-12z'/></svg></a>
                </div>
            </div>
        </div>
        
        <div class='member-card'>
            <img src='../members/megha.jpg' alt='Megha Gupta'>
            <h3>Megha Gupta</h3>
            <p class='role'>Doctoral Researcher</p>
            <p class='bio small-bio'>Megha leads the lab‚Äôs efforts in rare disease biomarker discovery, begiinning from ALS. She would be integrating integrating genomic, transcriptomic, and methylomic signals to decode early molecular signatures of neurodegeneration in ALS</p>
            <div class='member-links'><a class="mini-link" href="mailto:#"><svg viewBox='0 0 24 24'><path fill='currentColor'
d='M20 4H4a2 2 0 0 0-2 2v12a2 
2 0 0 0 2 2h16a2 2 0 0 0 
2-2V6a2 2 0 0 0-2-2zm0 4l-8 5-8-5V6l8 5 
8-5v2z'/></svg></a><a class="mini-link" href="#" target="_blank"><svg viewBox='0 0 24 24'><path fill='currentColor'
d='M19 3a2 2 0 0 1 2 2v14a2 2 0 0 
1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 
1 2-2h14M8 17v-7H6v7h2M7 
7.75A1.25 1.25 0 1 0 7 5a1.25 
1.25 0 0 0 0 2.75M18 
17v-4.5c0-2.25-2-3.25-3.5-3.25S12 
10.5 12 10.5V10H10v7h2v-4.2c0-1.25.75-1.8 
1.75-1.8S15.5 11.55 15.5 12.8V17h2z'/></svg></a></div>
        </div>
        
        <div class='member-card'>
            <img src='../members/download.jpg' alt='Ankit Patlan'>
            <h3>Ankit Patlan</h3>
            <p class='role'>Research Fellow</p>
            <p class='bio small-bio'>Ankit works on image informatics and building algorithms for automated interpretation of bone-marrow histopathology, which would help understand machine augmented interpretation, relevant for complimentary insights towards functional biology and trigger events in diseases.</p>
            <div class='member-links'><a class="mini-link" href="mailto:#"><svg viewBox='0 0 24 24'><path fill='currentColor'
d='M20 4H4a2 2 0 0 0-2 2v12a2 
2 0 0 0 2 2h16a2 2 0 0 0 
2-2V6a2 2 0 0 0-2-2zm0 4l-8 5-8-5V6l8 5 
8-5v2z'/></svg></a><a class="mini-link" href="#" target="_blank"><svg viewBox='0 0 24 24'><path fill='currentColor'
d='M19 3a2 2 0 0 1 2 2v14a2 2 0 0 
1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 
1 2-2h14M8 17v-7H6v7h2M7 
7.75A1.25 1.25 0 1 0 7 5a1.25 
1.25 0 0 0 0 2.75M18 
17v-4.5c0-2.25-2-3.25-3.5-3.25S12 
10.5 12 10.5V10H10v7h2v-4.2c0-1.25.75-1.8 
1.75-1.8S15.5 11.55 15.5 12.8V17h2z'/></svg></a></div>
        </div>
        
        <div class='member-card'>
            <img src='../members/kritika.jpg' alt='Kraticka Singhal'>
            <h3>Kraticka Singhal</h3>
            <p class='role'>Doctoral Researcher (Joint with PGIMER, Chd)</p>
            <p class='bio small-bio'>Kraticka is working on innovative multi-omics investigations, analyzing transcriptomic, proteomic, and metabolomic data in patients undergoing compromised heart function. Her investigation would aims to unveil biomarkers for Acute Coronary Syndrome and cardiac stress</p>
            <div class='member-links'><a class="mini-link" href="mailto:#"><svg viewBox='0 0 24 24'><path fill='currentColor'
d='M20 4H4a2 2 0 0 0-2 2v12a2 
2 0 0 0 2 2h16a2 2 0 0 0 
2-2V6a2 2 0 0 0-2-2zm0 4l-8 5-8-5V6l8 5 
8-5v2z'/></svg></a><a class="mini-link" href="#" target="_blank"><svg viewBox='0 0 24 24'><path fill='currentColor'
d='M19 3a2 2 0 0 1 2 2v14a2 2 0 0 
1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 
1 2-2h14M8 17v-7H6v7h2M7 
7.75A1.25 1.25 0 1 0 7 5a1.25 
1.25 0 0 0 0 2.75M18 
17v-4.5c0-2.25-2-3.25-3.5-3.25S12 
10.5 12 10.5V10H10v7h2v-4.2c0-1.25.75-1.8 
1.75-1.8S15.5 11.55 15.5 12.8V17h2z'/></svg></a></div>
        </div>
        
        <div class='member-card'>
            <img src='../members/paras.png' alt='Paras Verma, Ph.D.'>
            <h3>Paras Verma, Ph.D.</h3>
            <p class='role'>Graduate Researcher</p>
            <p class='bio small-bio'>Paras is working on germline cancer variant curation initiative, focusing on Indian population genetics. He is working on systematically integrates literature mining, genomic datasets, and FAIR-compliant pipelines to build variant repositories, that would enable comparative molecular and population-scale analyses.</p>
            <div class='member-links'><a class="mini-link" href="mailto:#"><svg viewBox='0 0 24 24'><path fill='currentColor'
d='M20 4H4a2 2 0 0 0-2 2v12a2 
2 0 0 0 2 2h16a2 2 0 0 0 
2-2V6a2 2 0 0 0-2-2zm0 4l-8 5-8-5V6l8 5 
8-5v2z'/></svg></a><a class="mini-link" href="#" target="_blank"><svg viewBox='0 0 24 24'><path fill='currentColor'
d='M19 3a2 2 0 0 1 2 2v14a2 2 0 0 
1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 
1 2-2h14M8 17v-7H6v7h2M7 
7.75A1.25 1.25 0 1 0 7 5a1.25 
1.25 0 0 0 0 2.75M18 
17v-4.5c0-2.25-2-3.25-3.5-3.25S12 
10.5 12 10.5V10H10v7h2v-4.2c0-1.25.75-1.8 
1.75-1.8S15.5 11.55 15.5 12.8V17h2z'/></svg></a></div>
        </div>
        
        <div class='member-card'>
            <img src='../members/keshav.jpg' alt='Keshav Kant'>
            <h3>Keshav Kant</h3>
            <p class='role'>Graduate Researcher</p>
            <p class='bio small-bio'>Keshav is working on blood‚Äìbrain barrier transporter modeling, where he would employ structural biology, ligand screening, and molecular dynamics to understand how transporters regulate metabolite movement across the BBB.</p>
            <div class='member-links'><a class="mini-link" href="mailto:#"><svg viewBox='0 0 24 24'><path fill='currentColor'
d='M20 4H4a2 2 0 0 0-2 2v12a2 
2 0 0 0 2 2h16a2 2 0 0 0 
2-2V6a2 2 0 0 0-2-2zm0 4l-8 5-8-5V6l8 5 
8-5v2z'/></svg></a><a class="mini-link" href="#" target="_blank"><svg viewBox='0 0 24 24'><path fill='currentColor'
d='M19 3a2 2 0 0 1 2 2v14a2 2 0 0 
1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 
1 2-2h14M8 17v-7H6v7h2M7 
7.75A1.25 1.25 0 1 0 7 5a1.25 
1.25 0 0 0 0 2.75M18 
17v-4.5c0-2.25-2-3.25-3.5-3.25S12 
10.5 12 10.5V10H10v7h2v-4.2c0-1.25.75-1.8 
1.75-1.8S15.5 11.55 15.5 12.8V17h2z'/></svg></a></div>
        </div>
        </div>
</section>

<section id="alumni">
  <h2 class="section-title">Alumni</h2>
  <p class="muted">Where our former members are today.</p>
  <div class="alumni-grid">
      <div class="alumni-card small">
        <img class="alumni-thumb" src="../members/nupur.jpg">
        <h4>Nupur Munjal, Ph.D.</h4>
        <p class="prev-role">Postdoctoral Fellow</p>
        <p class="duration">2023‚Äì2025</p>
        <p class="current-pos"><strong>Now:</strong> Asst. Prof. Amity Mohali</p>
      </div>
    
      <div class="alumni-card small">
        <img class="alumni-thumb" src="../members/muskan.jpg">
        <h4>Muskan Kumari</h4>
        <p class="prev-role">Research Fellow</p>
        <p class="duration">2024‚Äì2025</p>
        <p class="current-pos"><strong>Now:</strong> PhD Student, University of Helsinki</p>
      </div>
    
      <div class="alumni-card small">
        <img class="alumni-thumb" src="../members/akhil.jpg">
        <h4>Akhil Pratap</h4>
        <p class="prev-role">Research Fellow</p>
        <p class="duration">2023‚Äì2024</p>
        <p class="current-pos"><strong>Now:</strong> MD VKCAPITALS PVT. LTD.</p>
      </div>
    </div>
</section>

<section id="collaborations">
  <h2 class="section-title">Collaborations</h2>
  <p class="muted">Institutions partnering with MOSAIC</p>
  <div class="collab-grid">
      <div class="collab-card">
        <h3>PGIMER Chandigarh</h3>
        <p class="collab-desc">Studying Germline Variants for pediatric diseases and cardiac-mutiomics.</p>
        <a class="collab-link" href="https://pgimer.edu.in/" target="_blank">Visit ‚Üí</a>
      </div>
    
      <div class="collab-card">
        <h3>IISER Mohali</h3>
        <p class="collab-desc">Deep learning Imaging, Alternative splicing and structural modeling collaborations.</p>
        <a class="collab-link" href="https://www.iisermohali.ac.in/" target="_blank">Visit ‚Üí</a>
      </div>
    
      <div class="collab-card">
        <h3>Punjab University</h3>
        <p class="collab-desc">Hepcidin-Targeted Therapeutics for Anemia.</p>
        <a class="collab-link" href="https://puchd.ac.in/" target="_blank">Visit ‚Üí</a>
      </div>
    
      <div class="collab-card">
        <h3>Tata Memorial Cancer Center, New Chd.</h3>
        <p class="collab-desc">Upcoming collaboration, where synergy would be focused on finding unique molecular signatures in indian population for ther better treatment efficacy</p>
        <a class="collab-link" href="https://hbchs.tmc.gov.in/" target="_blank">Visit ‚Üí</a>
      </div>
    </div>
</section>

<section id="publications">
  <h2 class="section-title">Publications</h2>
  <div class="pub-list">
      <div class="pub-item">
        <h3 class="pub-title">Functional consequences of genetic variations in DgoR, a GntR/FadR family transcriptional repressor of D-galactonate metabolism in Escherichia coli</h3>
        <p class="pub-meta">Singh S, Arya G, Mishra R, Singla S, Pratap A, Upadhayay K, Sharma M, Chaba R ‚Ä¢ <strong class='pub-journal'>J Bacteriol.</strong> ‚Ä¢ 2025 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/40497739/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">Genetic variations in transcriptional regulators (TRs) of metabolic loci can influence host-bacterial interactions by affecting carbon utilization. Although the metabolism of sugar acids, including D-galactonate, is extensively implicated in the colonization and virulence of enteric bacteria, there has been no investigation on the extent of variations in their pathway-specific TRs. DgoR, the TR of D-galactonate metabolism, is the best-characterized GntR/FadR family sugar acid TR in enteric bacteria, recognized by the presence of an N-terminal winged helix-turn-helix DNA-binding domain and a C-terminal effector-binding and oligomerization (E-O) domain connected by a linker. Here, we examined 340 Escherichia coli isolates for variations in dgoR and studied their effect on repressor function. Genetic and biochemical studies identified variants with a partial loss of DNA-binding ability (P24L and A152E) and a decreased response to D-galactonate (R71C and P92L). Because the linker residue R71C resulted in a reduced response to D-galactonate and the E-O domain residue A152E led to a DNA binding defect, we performed simulations to probe their altered allosteric behavior. We observed that the correlation patterns, dynamics, and networks of the variants are indeed distinct from the wild type. Importantly, corroborating their repressor function, R71C and A152E variations impacted the growth of natural isolates in D-galactonate. Alignment-based variation detection across all E. coli and Enterobacterales identical protein group data sets revealed less prevalence of these four variations. Collectively, the present study highlights the need for a thorough analysis of the effect of variations in sugar acid TRs on repressor function and their effect on host-bacterial interactions.IMPORTANCESugar acids are used as carbon sources by enteric bacteria, both commensals and pathogens, with numerous studies highlighting their importance in host-bacterial interactions. Here, taking Escherichia coli DgoR, the transcriptional regulator (TR) of D-galactonate metabolism, as a representative, we showed that genetic variations in sugar acid TRs can affect their function and impact the utilization of these carbon sources by natural isolates. As the ability to use limiting nutrients enables bacteria to compete with the complex microbial community of the host, our study emphasizes the need for a comprehensive analysis of variations in sugar acid TRs to determine whether they influence the competition. These studies can help envision approaches for promoting the growth of commensals to eliminate their pathogenic counterparts.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Molecular mechanisms underlying allosteric behavior of Escherichia coli DgoR, a GntR/FadR family transcriptional regulator.</h3>
        <p class="pub-meta">Singh S, Arya G, Mishra R, Singla S, Pratap A, Upadhayay K, Sharma M, Chaba R ‚Ä¢ <strong class='pub-journal'>Nucleic Acids Res.</strong> ‚Ä¢ 2025 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/39777470/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">GntR/FadR family featuring an N-terminal winged helix-turn-helix DNA-binding domain and a C-terminal Œ±-helical effector-binding and oligomerization domain constitutes one of the largest families of transcriptional regulators. Several GntR/FadR regulators govern the metabolism of sugar acids, carbon sources implicated in bacterial-host interactions. Although effectors are known for a few sugar acid regulators, the unavailability of relevant structures has left their allosteric mechanism unexplored. Here, using DgoR, a transcriptional repressor of d-galactonate metabolism in Escherichia coli, as a model, and its superrepressor alleles, we probed allostery in a GntR/FadR family sugar acid regulator. Genetic and biochemical studies established compromised response to d-galactonate as the reason for the superrepressor behavior of the mutants: T180I does not bind d-galactonate, and while A97V, S171L and M188I bind d-galactonate, effector binding does not induce a conformational change required for derepression, suggesting altered allostery. For mechanistic insights into allosteric communication, we performed simulations of the modeled DgoR structure in different allosteric states for both the wild-type and mutant proteins. We found that each mutant exhibits unique dynamics disrupting the intrinsic allosteric communication pathways, thereby impacting DgoR function. We finally validated the allosteric communication model by testing in silico predictions with experimental data.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">In silico B-cell epitope prediction and molecular docking of Aspergillus allergens targeting improved ABPA diagnosis</h3>
        <p class="pub-meta">Agrawal D, Sharma M, Sachdeva R, Priyadarshini P, Minhas AP ‚Ä¢ <strong class='pub-journal'>J Asthma.</strong> ‚Ä¢ 2025 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/39425953/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">[Objective]: The objective of this study is to in silico predict Aspergillus fumigatus specific B-cell epitopes with a focus on enhancing Allergic Bronchopulmonary Aspergillosis (ABPA) diagnostic precision by using and to validate using molecular docking of Aspergillus fumigatus specific B-cell epitopes, aiming to overcome current serological and clinical method limitations and to support specific therapies and preventive strategies for better ABPA management. [Methods]: The sequences of Asp f1, Asp f2, Asp f3, and Asp f4 from NCBI were analyzed using IEDB-AR for B-cell epitope prediction. Structural modeling and molecular docking analysis were conducted using MODELER and HADDOCK, respectively, with visualization via PyMOL and PDBe PISA. [Results]: For Asp f1, two IgE-specific (40-47) and four IgG-specific (33-76, 125-148) B-cell epitopes were predicted. Asp f3 had one IgG-specific epitope (47-73), and Asp f4 had two IgG-specific epitopes (52-133) with no IgE epitopes. Asp f2 had eight IgE-specific epitopes (56-63, 93-99, 136-146, 153-160, 185-194, 200-206, 229-239) with IgPred scores above 0.931 and no IgG-specific epitopes. Molecular docking with HADDOCK Z-scores showed strong interactions between IgE and Asp f1 and Asp f2 epitopes. PyMOL and PISA-EBI identified key residues: LYS43 in Asp f1 forms a salt bridge with the IgE heavy chain. In Asp f2, out of nineteen identified residues, six residues (LYS 94, ARG 153, ASP 200, ASP 204, ASP 207 and GLU 233) were confirmed as part of the predicted IgE epitopes, exhibiting significant interactions with IgE, in agreement with both PyMOL and PISA analysis.[Conclusion]: This study aimed to enhance ABPA diagnostics by identifying key B-cell epitopes of Aspergillus fumigatus through in silico prediction and molecular docking, a way to support personalized therapies and preventive strategies in future.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Splicing of branchpoint-distant exons is promoted by Cactin, Tls1 and the ubiquitin-fold-activated Sde2</h3>
        <p class="pub-meta">Anupa T Anil; Karan Choudhary; Rakesh Pandian; Praver Gupta; Poonam Thakran; Arashdeep Singh; Monika Sharma; Shravan Kumar Mishra ‚Ä¢ <strong class='pub-journal'>Nucleic Acids Research</strong> ‚Ä¢ 2025 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/36095128/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">[Objective]: The objective of this study is to in silico predict Aspergillus fumigatus specific B-cell epitopes with a focus on enhancing Allergic Bronchopulmonary Aspergillosis (ABPA) diagnostic precision by using and to validate using molecular docking of Aspergillus fumigatus specific B-cell epitopes, aiming to overcome current serological and clinical method limitations and to support specific therapies and preventive strategies for better ABPA management. [Methods]: The sequences of Asp f1, Asp f2, Asp f3, and Asp f4 from NCBI were analyzed using IEDB-AR for B-cell epitope prediction. Structural modeling and molecular docking analysis were conducted using MODELER and HADDOCK, respectively, with visualization via PyMOL and PDBe PISA. [Results]: For Asp f1, two IgE-specific (40-47) and four IgG-specific (33-76, 125-148) B-cell epitopes were predicted. Asp f3 had one IgG-specific epitope (47-73), and Asp f4 had two IgG-specific epitopes (52-133) with no IgE epitopes. Asp f2 had eight IgE-specific epitopes (56-63, 93-99, 136-146, 153-160, 185-194, 200-206, 229-239) with IgPred scores above 0.931 and no IgG-specific epitopes. Molecular docking with HADDOCK Z-scores showed strong interactions between IgE and Asp f1 and Asp f2 epitopes. PyMOL and PISA-EBI identified key residues: LYS43 in Asp f1 forms a salt bridge with the IgE heavy chain. In Asp f2, out of nineteen identified residues, six residues (LYS 94, ARG 153, ASP 200, ASP 204, ASP 207 and GLU 233) were confirmed as part of the predicted IgE epitopes, exhibiting significant interactions with IgE, in agreement with both PyMOL and PISA analysis.[Conclusion]: This study aimed to enhance ABPA diagnostics by identifying key B-cell epitopes of Aspergillus fumigatus through in silico prediction and molecular docking, a way to support personalized therapies and preventive strategies in future.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Probing Nanoscale Lipid‚ÄìProtein Interactions at the Interface of Liquid Crystal Droplets</h3>
        <p class="pub-meta">Ipsita Pani; Fidha Nazreen K. M.; Monika Sharma; Santanu Kumar Pal ‚Ä¢ <strong class='pub-journal'>Nano Letters</strong> ‚Ä¢ 2025 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/34048245/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">[Objective]: The objective of this study is to in silico predict Aspergillus fumigatus specific B-cell epitopes with a focus on enhancing Allergic Bronchopulmonary Aspergillosis (ABPA) diagnostic precision by using and to validate using molecular docking of Aspergillus fumigatus specific B-cell epitopes, aiming to overcome current serological and clinical method limitations and to support specific therapies and preventive strategies for better ABPA management. [Methods]: The sequences of Asp f1, Asp f2, Asp f3, and Asp f4 from NCBI were analyzed using IEDB-AR for B-cell epitope prediction. Structural modeling and molecular docking analysis were conducted using MODELER and HADDOCK, respectively, with visualization via PyMOL and PDBe PISA. [Results]: For Asp f1, two IgE-specific (40-47) and four IgG-specific (33-76, 125-148) B-cell epitopes were predicted. Asp f3 had one IgG-specific epitope (47-73), and Asp f4 had two IgG-specific epitopes (52-133) with no IgE epitopes. Asp f2 had eight IgE-specific epitopes (56-63, 93-99, 136-146, 153-160, 185-194, 200-206, 229-239) with IgPred scores above 0.931 and no IgG-specific epitopes. Molecular docking with HADDOCK Z-scores showed strong interactions between IgE and Asp f1 and Asp f2 epitopes. PyMOL and PISA-EBI identified key residues: LYS43 in Asp f1 forms a salt bridge with the IgE heavy chain. In Asp f2, out of nineteen identified residues, six residues (LYS 94, ARG 153, ASP 200, ASP 204, ASP 207 and GLU 233) were confirmed as part of the predicted IgE epitopes, exhibiting significant interactions with IgE, in agreement with both PyMOL and PISA analysis.[Conclusion]: This study aimed to enhance ABPA diagnostics by identifying key B-cell epitopes of Aspergillus fumigatus through in silico prediction and molecular docking, a way to support personalized therapies and preventive strategies in future.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">A Comprehensive Analysis of cis-Acting RNA Elements in the SARS-CoV-2 Genome by a Bioinformatics Approach.</h3>
        <p class="pub-meta">Ipsita Pani; Fidha Nazreen K. M.; Monika Sharma; Santanu Kumar Pal ‚Ä¢ <strong class='pub-journal'>Frontiers in genetics</strong> ‚Ä¢ 2023 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/33424918/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">[Objective]: The objective of this study is to in silico predict Aspergillus fumigatus specific B-cell epitopes with a focus on enhancing Allergic Bronchopulmonary Aspergillosis (ABPA) diagnostic precision by using and to validate using molecular docking of Aspergillus fumigatus specific B-cell epitopes, aiming to overcome current serological and clinical method limitations and to support specific therapies and preventive strategies for better ABPA management. [Methods]: The sequences of Asp f1, Asp f2, Asp f3, and Asp f4 from NCBI were analyzed using IEDB-AR for B-cell epitope prediction. Structural modeling and molecular docking analysis were conducted using MODELER and HADDOCK, respectively, with visualization via PyMOL and PDBe PISA. [Results]: For Asp f1, two IgE-specific (40-47) and four IgG-specific (33-76, 125-148) B-cell epitopes were predicted. Asp f3 had one IgG-specific epitope (47-73), and Asp f4 had two IgG-specific epitopes (52-133) with no IgE epitopes. Asp f2 had eight IgE-specific epitopes (56-63, 93-99, 136-146, 153-160, 185-194, 200-206, 229-239) with IgPred scores above 0.931 and no IgG-specific epitopes. Molecular docking with HADDOCK Z-scores showed strong interactions between IgE and Asp f1 and Asp f2 epitopes. PyMOL and PISA-EBI identified key residues: LYS43 in Asp f1 forms a salt bridge with the IgE heavy chain. In Asp f2, out of nineteen identified residues, six residues (LYS 94, ARG 153, ASP 200, ASP 204, ASP 207 and GLU 233) were confirmed as part of the predicted IgE epitopes, exhibiting significant interactions with IgE, in agreement with both PyMOL and PISA analysis.[Conclusion]: This study aimed to enhance ABPA diagnostics by identifying key B-cell epitopes of Aspergillus fumigatus through in silico prediction and molecular docking, a way to support personalized therapies and preventive strategies in future.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Allosteric inhibition of MTHFR prevents futile SAM cycling and maintains nucleotide pools in one-carbon metabolism</h3>
        <p class="pub-meta">Muskan Bhatia, Jyotika Thakur, Shradha Suyal, Ruchika Oniel, Rahul Chakraborty, Shalini Pradhan, Monika Sharma, Shantanu Sengupta, Sunil Laxman, Shyam Kumar Masakapalli, Anand Kumar Bachhawat ‚Ä¢ <strong class='pub-journal'>J Biol Chem.</strong> ‚Ä¢ 2020 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/32934008/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">Methylenetetrahydrofolate reductase (MTHFR) links the folate cycle to the methionine cycle in one-carbon metabolism. The enzyme is known to be allosterically inhibited by SAM for decades, but the importance of this regulatory control to one-carbon metabolism has never been adequately understood. To shed light on this issue, we exchanged selected amino acid residues in a highly conserved stretch within the regulatory region of yeast MTHFR to create a series of feedback-insensitive, deregulated mutants. These were exploited to investigate the impact of defective allosteric regulation on one-carbon metabolism. We observed a strong growth defect in the presence of methionine. Biochemical and metabolite analysis revealed that both the folate and methionine cycles were affected in these mutants, as was the transsulfuration pathway, leading also to a disruption in redox homeostasis. The major consequences, however, appeared to be in the depletion of nucleotides. 13C isotope labeling and metabolic studies revealed that the deregulated MTHFR cells undergo continuous transmethylation of homocysteine by methyltetrahydrofolate (CH3THF) to form methionine. This reaction also drives SAM formation and further depletes ATP reserves. SAM was then cycled back to methionine, leading to futile cycles of SAM synthesis and recycling and explaining the necessity for MTHFR to be regulated by SAM. The study has yielded valuable new insights into the regulation of one-carbon metabolism, and the mutants appear as powerful new tools to further dissect out the intersection of one-carbon metabolism with various pathways both in yeasts and in humans.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Investigating the pathogenic SNPs in BLM helicase and their biological consequences by computational approach</h3>
        <p class="pub-meta">Faisal A Alzahrani#, Firoz Ahmed#, Monika Sharma#, Mohd Rehan, Maryam Mahfuz, Mohammed N Baeshen, Yousef Hawsawi, Ahmed Almatrafi, Suliman Abdallah Alsagaby, Mohammad Azhar Kamal, Mohiuddin Khan Warsi, Hani Choudhry, Mohammad Sarwar Jamal ‚Ä¢ <strong class='pub-journal'>Sci Rep.</strong> ‚Ä¢ 2020 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/32704157/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">The BLM helicase protein plays a vital role in DNA replication and the maintenance of genomic integrity. Variation in the BLM helicase gene resulted in defects in the DNA repair mechanism and was reported to be associated with Bloom syndrome (BS) and cancer. Despite extensive investigation of helicase proteins in humans, no attempt has previously been made to comprehensively analyse the single nucleotide polymorphism (SNPs) of the BLM gene. In this study, a comprehensive analysis of SNPs on the BLM gene was performed to identify, characterize and validate the pathogenic SNPs using computational approaches. We obtained SNP data from the dbSNP database version 150 and mapped these data to the genomic coordinates of the &#x27;NM_000057.3&#x27; transcript expressing BLM helicase (P54132). There were 607 SNPs mapped to missense, 29 SNPs mapped to nonsense, and 19 SNPs mapped to 3&#x27;-UTR regions. Initially, we used many consensus tools of SIFT, PROVEAN, Condel, and PolyPhen-2, which together increased the accuracy of prediction and identified 18 highly pathogenic non-synonymous SNPs (nsSNPs) out of 607 SNPs. Subsequently, these 18 high-confidence pathogenic nsSNPs were analysed for BLM protein stability, structure-function relationships and disease associations using various bioinformatics tools. These 18 mutants of the BLM protein along with the native protein were further investigated using molecular dynamics simulations to examine the structural consequences of the mutations, which might reveal their malfunction and contribution to disease. In addition, 28 SNPs were predicted as &#x27;stop gained&#x27; nonsense SNPs and one SNP was predicted as &#x27;start lost&#x27;. Two SNPs in the 3&#x27;UTR were found to abolish miRNA binding and thus may enhance the expression of BLM. Interestingly, we found that BLM mRNA overexpression is associated with different types of cancers. Further investigation showed that the dysregulation of BLM is associated with poor overall survival (OS) for lung and gastric cancer patients and hence led to the conclusion that BLM has the potential to be used as an important prognostic marker for the detection of lung and gastric cancer.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">In silico characterization of residues essential for substrate binding of human cystine transporter, xCT</h3>
        <p class="pub-meta">Monika Sharma, C R Anirudh ‚Ä¢ <strong class='pub-journal'>J Mol Model.</strong> ‚Ä¢ 2019 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/31705320/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">xCT is a sodium-independent amino acid antiporter that imports L-cystine and exports L-glutamate in a 1:1 ratio. It is a component of heterodimeric amino acid transporter system Xc- working at the cross-roads of maintaining neurological processes and regulating antioxidant defense. The transporter has 12 transmembrane domains with intracellular N- and C-termini, and like other transporter proteins can undergo various conformational changes while switching the ligand accessibilities from intracellular to extracellular site. In the present study, we generated two homology models of human xCT in two distinct conformations: inward-facing occluded state and outward-facing open state. Our results indicated the substrate translocation channel composed of transmembrane helices TMs 1, 3, 6, 8, and 10. We docked anionic L-cystine and L-glutamate within the cavities to assess the two distinct binding scenarios for xCT as antiporter. We also assessed the interactions between the ligands and transporter and observed that ligands bind to similar residues within the channel. Using MM-PBSA/MM-GBSA approach, we computed the binding energies of these ligands to different conformational states. Cystine and glutamate bind xCT with favorable binding energies, with more favorable binding observed in inward occluded state than in outward open state. We further computed the residue-wise decomposition of these binding energies and identified the residues as essential for substrate binding/permeation. Filtering the residues that form favorable energetic contributions to the ligand binding in both the states, our studies suggest T56, A60, R135, A138, V141, Y244, A247, F250, S330, L392, and R396 as critical residues for ligand binding as well as ligand transport for any conformational state adopted by xCT during its transport cycle.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Understanding the binding specificities of mRNA targets by the mammalian Quaking protein</h3>
        <p class="pub-meta">Monika Sharma, Shakshi Sharma, Apoorv Alawada ‚Ä¢ <strong class='pub-journal'>Nucleic Acids Res.</strong> ‚Ä¢ 2019 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/31602485/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">Mammalian Quaking (QKI) protein, a member of STAR family of proteins is a mRNA-binding protein, which post-transcriptionally modulates the target RNA. QKI protein possesses a maxi-KH domain composed of single heterogeneous nuclear ribonucleoprotein K homology (KH) domain and C-terminal QUA2 domain, that binds a sequence-specific QKI RNA recognition element (QRE), CUAAC. To understand the binding specificities for different mRNA sequences of the KH-QUA2 domain of QKI protein, we introduced point mutations at different positions in the QRE resulting in twelve different mRNA sequences with single nucleotide change. We carried out long unbiased molecular dynamics simulations using two different sets of recently updated forcefield parameters: AMBERff14SB+RNAœáOL3 and CHARMM36 (with CMAP correction). We analyzed the changes in intermolecular dynamics as a result of mutation. Our results show that AMBER forcefields performed better to model the interactions between mRNA and protein. We also calculated the binding affinities of different mRNA sequences and found that the relative order correlates to the reported experimental studies. Our study shows that the favorable binding with the formation of stable complex will occur when there is an increase of the total intermolecular contacts between mRNA and protein, but without the loss of native contacts within the KH-QUA domain.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Design and synthesis of sulfur cross-linked 1,3,4-oxadiazole-nitro(furan/thiophene)-propenones as dual inhibitors of inflammation and tuberculosis: molecular docking and Hirshfeld surface analysis</h3>
        <p class="pub-meta">Vishwanath Turukarabettu, Balakrishna Kalluraya &amp; Monika Sharma ‚Ä¢ <strong class='pub-journal'>Monatsh Chem.</strong> ‚Ä¢ 2019 <a class="pub-link" href="https://link.springer.com/article/10.1007/s00706-019-02507-2" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">A series of 3-[5-nitro(furan/thiophene)-2-yl]-1-aryl-3-(5-aryl-1,3,4-oxadiazol-2-ylthio)prop-2-en-1-one derivatives was synthesized and studied with the aim of developing dual inhibitors of multidrug-resistant tuberculosis and inflammation. The in vivo anti-inflammatory activity results showed excellent inhibition of rat paw edema. The methoxybenzene/nitrofuryl derivative of title compounds showed 83% inhibition of inflammation during 2‚Äì6 h after carrageenan injection. All compounds showed anti-tuberculosis activity at MIC of 50 ¬µg/cm3. The molecular docking studies revealed that the oxadiazole and nitrofuran groups played a significant role in the inhibiting site of the enzymes COX1, COX2, 5-LOs, and InhA by forming hydrogen bonding with Tyr 385, Ser 530, Tyr 467, and Tyr 158 amino acid residues, respectively. The novel compounds are active antibacterial agents with potential inhibition on E. coli bacteria. The toxicity results showed good percentage viability of human kidney cell lines with IC50 value greater than 100 ¬µg/cm3 concentration. The Hirshfeld surface analysis and electrostatic potential map of compound showed good intermolecular contacts and hydrogen bonding donor and acceptor potential.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Unfolding Transitions of Peripheral Subunit Binding Domains Show Cooperative Behavior</h3>
        <p class="pub-meta">Monika Sharma, Gopalakrishnan Bulusu, Abhijit Mitra ‚Ä¢ <strong class='pub-journal'>J Phys Chem B.</strong> ‚Ä¢ 2019 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/30958002/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">Characterization of native, intermediate, and denatured states is crucial for understanding the factors influencing the stability of proteins. We have carried out molecular dynamics simulations to study the unfolding of three peripheral subunit binding domains (PSBDs): E. coli BBL, Bacillus stearothermophilus E3BD, and human hbSBD, at three different temperatures: 300, 330, and 400 K, and in the presence of two solvents: water and 5 M guanidinium hydrochloride (GndCl) solution. These proteins share similar folds, with two parallel helices, maintained via a hydrophobic core comprising residues from their interconnecting loop. BBL is more sensitive to thermal and chemical denaturation in comparison to hbSBD, and E3BD is the most stable of all of the three proteins. The effect of temperature on the stability of these proteins is more pronounced in &#x27;water-only&#x27; simulations compared to that in the presence of guanidium hydrochloride in high concentrations. Our results show cooperative unfolding transitions of these proteins, which are triggered by an initial melting of the C-terminal helix H2. The consequent loss of interhelical interactions or native contacts, as observed, leads to the subsequent melting of the N-terminal helix H1.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Interpretation of spectroscopic data using molecular simulations for the secondary active transporter BetP</h3>
        <p class="pub-meta">Vanessa Leone, Izabela Waclawska, Katharina Kossmann, Caroline Koshy, Monika Sharma, Thomas F Prisner, Christine Ziegler,  Burkhard Endeward, Lucy R Forrest ‚Ä¢ <strong class='pub-journal'>J Gen Physiol.</strong> ‚Ä¢ 2019 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/30728216/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">Mechanistic understanding of dynamic membrane proteins such as transporters, receptors, and channels requires accurate depictions of conformational ensembles, and the manner in which they interchange as a function of environmental factors including substrates, lipids, and inhibitors. Spectroscopic techniques such as electron spin resonance (ESR) pulsed electron-electron double resonance (PELDOR), also known as double electron-electron resonance (DEER), provide a complement to atomistic structures obtained from x-ray crystallography or cryo-EM, since spectroscopic data reflect an ensemble and can be measured in more native solvents, unperturbed by a crystal lattice. However, attempts to interpret DEER data are frequently stymied by discrepancies with the structural data, which may arise due to differences in conditions, the dynamics of the protein, or the flexibility of the attached paramagnetic spin labels. Recently, molecular simulation techniques such as EBMetaD have been developed that create a conformational ensemble matching an experimental distance distribution while applying the minimal possible bias. Moreover, it has been proposed that the work required during an EBMetaD simulation to match an experimentally determined distribution could be used as a metric with which to assign conformational states to a given measurement. Here, we demonstrate the application of this concept for a sodium-coupled transport protein, BetP. Because the probe, protein, and lipid bilayer are all represented in atomic detail, the different contributions to the work, such as the extent of protein backbone movements, can be separated. This work therefore illustrates how ranking simulations based on EBMetaD can help to bridge the gap between structural and biophysical data and thereby enhance our understanding of membrane protein conformational mechanisms.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Atomic-level characterization of conformational transition and substrate binding of xCT transporter</h3>
        <p class="pub-meta">M. Sharma, A. C. Rohithaswa ‚Ä¢ <strong class='pub-journal'>bioRxiv</strong> ‚Ä¢ 2018 <a class="pub-link" href="https://www.biorxiv.org/content/10.1101/389643v1" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">xCT is a component of heterodimeric amino acids transporter system Xc- that has been known to work at the cross-roads of maintaining neurological processes and regulating antioxidant defense. xCT is a sodium-independent amino acid antiporter, that imports L- cystine and exports L-glutamate in a 1:1 ratio. The transporter has 12 transmembrane domains with intracellular N- and C-termini, which can undergo various conformational changes while switching the ligand accessibilities from intracellular to extracellular site. In the present study, we generated two homology models of human xCT in two distinct conformations: inward facing occluded state and outward facing open state. We investigated the conformational transitions within these two states by employing series of targeted molecular dynamics simulations. Our results indicated the substrate translocation channel composed of transmembrane helices TMs 1, 3, 6, 8, and 10. Further, we analyzed the ligand binding within the intermediate conformations obtained from the transition simulations. We docked anionic L-cystine and L-glutamate within the cavities alone or in combination to assess the two distinct binding scenarios for xCT as antiporter. We also assessed the interactions between the ligand and xCT and observed that ligands bind to similar residues within the channel, and these residues are essential for substrate binding/permeation. In addition, we analyzed the correlations between ligand binding and conformational transition and observed conformations that are representatives for intermediate ligand bound states. The results presented in the study provide insights into the interplay of conformational transition and ligand binding as xCT goes from one probable conformation to another while transporting the ligand. And the data thus adds to the existing evidence of alternating access mechanism pertaining to the functioning of transporters.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Insights into the molecular basis for substrate binding and specificity of the fungal cystine transporter CgCYN1</h3>
        <p class="pub-meta">Anup Arunrao Deshpande, Monika Sharma, Anand Kumar Bachhawat ‚Ä¢ <strong class='pub-journal'>Biochim Biophys Acta Biomembr.</strong> ‚Ä¢ 2017 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/28865795/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">Cystine transporters are a clinically important class of transporters found in bacteria, pathogenic fungi and mammalian cells. Despite their significance, very little is known about the mechanism of substrate recognition and transport. We have carried out studies on the plasma membrane Candida glabrata cystine transporter, CgCYN1 a member of the amino acid-polyamine-organocation (APC) transporter superfamily. A homology model of CgCYN1 was generated by using crystal structures of three known bacterial APC transporters followed by further refinement using molecular dynamics simulations. This revealed a possible translocation channel lined by TMD1, TMD3, TMD6, TMD8 and TMD10 helices. In silico docking studies with cystine along with comparison with other known cystine permeases and closely related lysine permeases allowed prediction of amino acid residues specifically involved in cystine binding. To validate this model a total of 19 predicted residues were subjected to site directed mutagenesis and functionally evaluated by growth on cystine and the analogues cystathionine and seleno-dl-cystine. Biochemical evaluation by radioactive uptake assays confirmed that these mutants showed reduced cystine uptake. Detailed kinetic analysis studies for the transport defective mutants revealed the involvement of residue G255 from the conserved FAYGGTE motif of TMD 6, and T339, S340 and H347 (all from TMD 8) in cystine binding. The implications of these findings on the homologous mammalian cystine transporter, XcT are also discussed.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Mechanism of mRNA-STAR domain interaction: Molecular dynamics simulations of Mammalian Quaking STAR protein</h3>
        <p class="pub-meta">Monika Sharma, C R Anirudh ‚Ä¢ <strong class='pub-journal'>Sci Rep.</strong> ‚Ä¢ 2017 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/28974714/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">STAR proteins are evolutionary conserved mRNA-binding proteins that post-transcriptionally regulate gene expression at all stages of RNA metabolism. These proteins possess conserved STAR domain that recognizes identical RNA regulatory elements as YUAAY. Recently reported crystal structures show that STAR domain is composed of N-terminal QUA1, K-homology domain (KH) and C-terminal QUA2, and mRNA binding is mediated by KH-QUA2 domain. Here, we present simulation studies done to investigate binding of mRNA to STAR protein, mammalian Quaking protein (QKI). We carried out conventional MD simulations of STAR domain in presence and absence of mRNA, and studied the impact of mRNA on the stability, dynamics and underlying allosteric mechanism of STAR domain. Our unbiased simulations results show that presence of mRNA stabilizes the overall STAR domain by reducing the structural deviations, correlating the &#x27;within-domain&#x27; motions, and maintaining the native contacts information. Absence of mRNA not only influenced the essential modes of motion of STAR domain, but also affected the connectivity of networks within STAR domain. We further explored the dissociation of mRNA from STAR domain using umbrella sampling simulations, and the results suggest that mRNA binding to STAR domain occurs in multi-step: first conformational selection of mRNA backbone conformations, followed by induced fit mechanism as nucleobases interact with STAR domain.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Long-Range Signaling in MutS and MSH Homologs via Switching of Dynamic Communication Pathways</h3>
        <p class="pub-meta">Beibei Wang, Joshua Francis, Monika Sharma, Sean M Law, Alexander V Predeus, Michael Feig ‚Ä¢ <strong class='pub-journal'>PLoS Comput Biol.</strong> ‚Ä¢ 2016 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/27768684/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">Allostery is conformation regulation by propagating a signal from one site to another distal site. This study focuses on the long-range communication in DNA mismatch repair proteins MutS and its homologs where intramolecular signaling has to travel over 70 √Ö to couple lesion detection to ATPase activity and eventual downstream repair. Using dynamic network analysis based on extensive molecular dynamics simulations, multiple preserved communication pathways were identified that would allow such long-range signaling. The pathways appear to depend on the nucleotides bound to the ATPase domain as well as the type of DNA substrate consistent with previously proposed functional cycles of mismatch recognition and repair initiation by MutS and homologs. A mechanism is proposed where pathways are switched without major conformational rearrangements allowing for efficient long-range signaling and allostery.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Conformational preferences of DNA in reduced dielectric environments</h3>
        <p class="pub-meta">Asli Yildirim, Monika Sharma, Bradley Michael Varner, Liang Fang, Michael Feig ‚Ä¢ <strong class='pub-journal'>J Phys Chem B.</strong> ‚Ä¢ 2014 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/25166278/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">The effect of reduced dielectric environments on the conformational sampling of DNA was examined through molecular dynamics simulations. Different dielectric environments were used to model one aspect of cellular environments. Implicit solvent based on the Generalized Born methodology was used to reflect different dielectric environments in the simulations. The simulation results show a tendency of DNA structures to favor noncanonical A-like conformations rather than canonical A- and B-forms as a result of the reduced dielectric environments. The results suggest that the reduced dielectric response in cellular environments may be sufficient to enhance the sampling of A-like DNA structures compared to dilute solvent conditions.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">A designed conformational shift to control protein binding specificity</h3>
        <p class="pub-meta">Servaas Michielssens, Jan Henning Peters, David Ban, Supriya Pratihar, Daniel Seeliger, Monika Sharma, Karin Giller, Thomas Michael Sabo, Stefan Becker, Donghan Lee, Christian Griesinger, Bert L de Groot ‚Ä¢ <strong class='pub-journal'>Angew Chem Int Ed Engl.</strong> ‚Ä¢ 2014 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/25115701/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">In a conformational selection scenario, manipulating the populations of binding-competent states should be expected to affect protein binding. We demonstrate how in silico designed point mutations within the core of ubiquitin, remote from the binding interface, change the binding specificity by shifting the conformational equilibrium of the ground-state ensemble between open and closed substates that have a similar population in the wild-type protein. Binding affinities determined by NMR titration experiments agree with the predictions, thereby showing that, indeed, a shift in the conformational equilibrium enables us to alter ubiquitin&#x27;s binding specificity and hence its function. Thus, we present a novel route towards designing specific binding by a conformational shift through exploiting the fact that conformational selection depends on the concentration of binding-competent substates.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Differential Mismatch Recognition Specificities of Eukaryotic MutS Homologs, MutSŒ± and MutSŒ≤</h3>
        <p class="pub-meta">Monika Sharma, Alexander V Predeus, Shayantani Mukherjee, Michael Feig ‚Ä¢ <strong class='pub-journal'>Biophys J.</strong> ‚Ä¢ 2014 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/24896128/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">In eukaryotes, the recognition of the DNA postreplication errors and initiation of the mismatch repair is carried out by two MutS homologs: MutSŒ± and MutSŒ≤. MutSŒ± recognizes base mismatches and 1 to 2 unpaired nucleotides whereas MutSŒ≤ recognizes longer insertion-deletion loops (IDLs) with 1 to 15 unpaired nucleotides as well as certain mismatches. Results from molecular dynamics simulations of native MutSŒ≤:IDL-containing DNA and MutSŒ±:mismatch DNA complexes as well as complexes with swapped DNA substrates provide mechanistic insight into how the differential substrate specificities are achieved by MutSŒ± and MutSŒ≤, respectively. Our simulations results suggest more extensive interactions between MutSŒ≤ and IDL-DNA and between MutSŒ± and mismatch-containing DNA that suggest corresponding differences in stability. Furthermore, our simulations suggest more expanded mechanistic details involving a different degree of bending when DNA is bound to either MutSŒ± or MutSŒ≤ and a more likely opening of the clamp domains when noncognate substrates are bound. The simulation results also provide detailed information on key residues in MutSŒ≤ and MutSŒ± that are likely involved in recognizing IDL-DNA and mismatch-containing DNA, respectively.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">DNA bending propensity in the presence of base mismatches: implications for DNA repair</h3>
        <p class="pub-meta">Monika Sharma, Alexander V Predeus, Shayantani Mukherjee, Michael Feig ‚Ä¢ <strong class='pub-journal'>J Phys Chem B.</strong> ‚Ä¢ 2013 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/23621762/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">DNA bending is believed to facilitate the initial recognition of the mismatched base for repair. The repair efficiencies are dependent on both the mismatch type and neighboring nucleotide sequence. We have studied bending of several DNA duplexes containing canonical matches: A:T and G:C; various mismatches: A:A, A:C, G:A, G:G, G:T, C:C, C:T, and T:T; and a bis-abasic site: X:X. Free-energy profiles were generated for DNA bending using umbrella sampling. The highest energetic cost associated with DNA bending is observed for canonical matches while bending free energies are lower in the presence of mismatches, with the lowest value for the abasic site. In all of the sequences, DNA duplexes bend toward the major groove with widening of the minor groove. For homoduplexes, DNA bending is observed to occur via smooth deformations, whereas for heteroduplexes, kinks are observed at the mismatch site during strong bending. In general, pyrimidine:pyrimidine mismatches are the most destabilizing, while purine:purine mismatches lead to intermediate destabilization, and purine:pyrimidine mismatches are the least destabilizing. The ease of bending is partially correlated with the binding affinity of MutS to the mismatch pairs and subsequent repair efficiencies, indicating that intrinsic DNA bending propensities are a key factor of mismatch recognition.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Carboxylated Lysine is Required for Higher Activities in Hydantoinases</h3>
        <p class="pub-meta">Vijay Kumar, Neetu Saxena, Monika Sarma, K V Radha Kishan ‚Ä¢ <strong class='pub-journal'>Protein Pept Lett.</strong> ‚Ä¢ 2011 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/21413921/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">Hydantoinases are industrial enzymes with varying degree of activities on variable substrates to form different products. Although, few of the hydantoinase structures were known recently, the functional details and active site mechanism were not clearly understood yet. In a structure determination effort we reported that Bacillus sp. AR9 hydantoinase contains uncarboxylated lysine in the active site, whereas all the other hydantoinases have a carboxylated active site lysine. Here we describe the importance of carboxylated lysine for differential activities by making lysine mutations as well as carboxylating the lysine in a D-hydantoinase from Bacillus sp. AR9. The lysine to alanine and lysine to arginine mutations showed reduced activities whereas carboxylation of the lysine has enhanced the activity. Theoretical studies involving the calculation of electrostatic potentials for the hydroxide ion between the two metal ions present in the active site suggest that the presence of carboxylated lysine increases the nucleophilicity of the hydroxide.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Comparative modeling of thioredoxin glutathione reductase from Schistosoma mansoni: A multifunctional target for antischistosomal therapy</h3>
        <p class="pub-meta">Monika Sharma, Smriti Khanna, Gopalakrishnan Bulusu, Abhijit Mitra ‚Ä¢ <strong class='pub-journal'>J Mol Graph Model</strong> ‚Ä¢ 2009 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/19070522/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">Schistosoma mansoni, a trematode parasite, which causes schistosomiasis and affects more than 200 million people worldwide, lives in an aerobic environment and therefore needs an effective redox mechanism for surviving reactive oxygen species from its host. Although, the host has two different redox systems: glutaredoxin and thioredoxin, the parasite has only one unique multifunctional enzyme, thioredoxin glutathione reductase (TGR) involving a fusion of two proteins, glutaredoxin (Grx) and thioredoxin reductase (TR), for performing all the redox activities. This dependence of S. mansoni on a single protein, TGR, for its protection from oxidative stress, makes it a promising drug target. Here, we describe a suitably validated, homology model for S. mansoni TGR (SmTGR), developed using both TR and Grx templates, functionally complete in the dimeric form with cofactors NADP(H) and FAD. Comparative analysis of substrate and inhibitor binding pockets of our model with crystal structures of parent TR as well as with that of glutathione reductase (GR), which is an essential component of the Grx system, appears to provide greater insight into the functioning of TGR. This also augments recent observations reported on the basis of X-ray structure data on SmTGR monomer lacking the C-terminal selenocysteine tail.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">MD simulations of ligand-bound and ligand-free aptamer: molecular level insights into the binding and switching mechanism of the add A-riboswitch</h3>
        <p class="pub-meta">Monika Sharma, Gopalakrishnan Bulusu, Abhijit Mitra ‚Ä¢ <strong class='pub-journal'>RNA</strong> ‚Ä¢ 2009 <a class="pub-link" href="https://pubmed.ncbi.nlm.nih.gov/19625387/" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">Riboswitches are structural cis-acting genetic regulatory elements in 5&#x27; UTRs of mRNAs, consisting of an aptamer domain that regulates the behavior of an expression platform in response to its recognition of, and binding to, specific ligands. While our understanding of the ligand-bound structure of the aptamer domain of the adenine riboswitches is based on crystal structure data and is well characterized, understanding of the structure and dynamics of the ligand-free aptamer is limited to indirect inferences from physicochemical probing experiments. Here we report the results of 15-nsec-long explicit-solvent molecular dynamics simulations of the add A-riboswitch crystal structure (1Y26), both in the adenine-bound (CLOSED) state and in the adenine-free (OPEN) state. Root-mean-square deviation, root-mean-square fluctuation, dynamic cross-correlation, and backbone torsion angle analyses are carried out on the two trajectories. These, along with solvent accessible surface area analysis of the two average structures, are benchmarked against available experimental data and are shown to constitute the basis for obtaining reliable insights into the molecular level details of the binding and switching mechanism. Our analysis reveals the interaction network responsible for, and conformational changes associated with, the communication between the binding pocket and the expression platform. It further highlights the significance of a, hitherto unreported, noncanonical W:H trans base pairing between A73 and A24, in the OPEN state, and also helps us to propose a possibly crucial role of U51 in the context of ligand binding and ligand discrimination.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Chemical oscillations: Mathematical modelling</h3>
        <p class="pub-meta">Monika Sharma, Praveen Kumar ‚Ä¢ <strong class='pub-journal'>Reson.</strong> ‚Ä¢ 2006 <a class="pub-link" href="https://link.springer.com/article/10.1007/BF02835994" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">The law of mass-action led chemists to the belief that reactions approach equilibrium steadily. So the discovery of chemical oscillations came as a surprise. Now chemists are very familiar with reactions that oscillate in time and/or space. Experimental and theoretical studies of such reactions showing temporal and spatial oscillations attract the interest of many laboratories world-wide. The Lotka-Volterra model is the simplest mathematical model which exhibits such oscillations. In this article, we use this model to illustrate chemical oscillations with the help of a computer program.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Chemical oscillations: Basic principles and examples</h3>
        <p class="pub-meta">Monika Sharma &amp; Praveen Kumar ‚Ä¢ <strong class='pub-journal'>Reson.</strong> ‚Ä¢ 2006 <a class="pub-link" href="https://link.springer.com/article/10.1007/BF02837274" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">A chemical reaction is usually thought of as coming together of reactant molecules to form products. The concentrations of initial components (reactants) decrease, and concentrations of products increase until they reach a well defined state: the equilibrium. This process is accompanied by a decrease of the system free energy (compared at constant pressure and temperature), until it reaches a minimum in the equilibrium. Thus, it follows from the nature of the law of mass-action that every simple reaction approaches its equilibrium asymptotically, and the evolution of any physico-chemical system leads invariably to the steady state of maximum disorder in the universe. Normally, chemical systems approach equilibrium in a smooth, frequently exponential relaxation. Under special circumstances, however, coherent behavior such as sustained oscillations are observed and the oscillations of chemical origin have been present as long as life itself. Such reactions can be studied using mathematical models, the Lotka-Volterra model being the earliest and the simplest one.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Graph theoretic approach for studying correlated motions in biomolecules</h3>
        <p class="pub-meta">Eshita Mutt; Monika Sharma; Abhijit Mitra; Jyothish Soman; Kothapalli Kishore; Naveena Yanamala ‚Ä¢ <strong class='pub-journal'>2009 World Congress on Nature &amp; Biologically Inspired Computing (NaBIC)</strong> ‚Ä¢ 2009 <a class="pub-link" href="https://ieeexplore.ieee.org/document/5393610" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">The paper describes the application of graph theoretic concepts to the dynamic cross-correlation data obtained from MD simulations of adenine riboswitch, in the absence and presence of adenine. This novel approach combines both community detection algorithms that support edge weights, and cliques. The effect of variations in the values of nearest neighbors (NN) and correlation coefficient threshold (T) in the community detection algorithm have been applied to identify and filter out coincidental correlations between rogue nodes. The results generated for add Adenine riboswitch based on this hybrid approach, successfully identified the correlations within the structural regions of the molecule, providing strong clues regarding the functionality and stability of the RNA molecule in the absence and presence of adenine. Our results also suggested that a prior application of the proposed algorithm (in an automated fashion) to the simulation data of RNA biomolecules, can provide strong leads for hypothesis formulation and subsequent hypothesis-driven manual investigation.</p>
      </div>
    
      <div class="pub-item">
        <h3 class="pub-title">Preferential cyclotrimerization of 5-(aminomethyl)-2-furancarboxylic acid (AMFC): Electrostatic and orbital interactions studies</h3>
        <p class="pub-meta">Monika Sharma, Praveen Kumar, Harjinder Singh, Tushar K. Chakraborty ‚Ä¢ <strong class='pub-journal'>Journal of Molecular Structure: THEOCHEM</strong> ‚Ä¢ 2006 <a class="pub-link" href="https://doi.org/10.1016/j.theochem.2006.02.010" target="_blank">[View]</a></p>
        <button class="abstract-toggle">Show Abstract ‚ñæ</button>
        <p class="abstract-text" style="display:none;">An attempt is made to develop a theoretical understanding beyond a phenomenological observation of the preferential cyclotrimerization of 5-(aminomethyl)-2-furancarboxylic acid (AMFC). Such cyclopeptides are of great interest due to their considerable bioactivity. A detailed analysis of the molecular properties such as electrostatic potentials, and electron densities is carried out. The electrostatic potentials (ESP) in the region around molecular skeletons are investigated to provide justification for the preferential formation of the cyclic tripeptide over the cyclic dipeptide from the building block, AMFC. The ESP are calculated and computed on the molecular surfaces of the oligopeptides. The surfaces are provided by the 0.002-electrons/bohr3 contour of the total electronic density function. There are repulsions observed in the dimer due to the proximity of the ethereal oxygen atoms of the furan rings. The possibility of counter-balancing the strain observed in rings by the presence of O‚ãØH‚ÄìN hydrogen bonds in the oligopetides is also investigated. Natural bond orbital (NBO) analysis is done to find the second-order interactions, and atoms in molecules (AIM) calculations are performed to explore the interactions between the donor and acceptor moieties in the molecule.</p>
      </div>
    </div>
</section>

<section id="join">
  <h2 class="section-title">Join MOSAIC</h2>
  <p class="muted">We welcome structural, omics, and imaging researchers.</p>
  <div class="cta-row">
    <a class="btn" href="mailto:msplaksha@gmail.com">Email msplaksha@gmail.com</a>
  </div>
</section>

</div>
</body>
</html>

